extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -406,6 +404,78 @@ angular.module('oppia').component('svgFilenameEditor', {
         };
       };
 
+      var loadGroupedObject = function(objId, obj, groupedObjects) {
+        // Checks the first five characters of the id.
+        if (objId.slice(0, 5) === 'group') {
+          if (groupedObjects.length <= objId.slice(5)) {
+            groupedObjects.push([]);
+          }
+          obj.toSVG = ctrl.createCustomToSVG(
+            obj.toSVG, obj.type, obj.id);
+          groupedObjects[objId.slice(5)].push(obj);
```

Comment:
`objId.slice(5)` seems repeated. Let's just do it once at the top and store in a variable?

Reply:
Done.